com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -2123,6 +2154,44 @@ private void SyncTime()
 #endif
         }
 
+        internal struct DisconnectReasonMessage : INetworkMessage
+        {
+            public string Reason;
+
+            public void Serialize(FastBufferWriter writer)
+            {
+                if (writer.TryBeginWrite(sizeof(int) + FastBufferWriter.GetWriteSize(Reason)))
+                {
+                    writer.WriteValue(Reason.Length);
+                    writer.WriteValue(Reason);
+                }
+                else
+                {
+                    writer.TryBeginWrite(sizeof(int) + FastBufferWriter.GetWriteSize(""));
+                    writer.WriteValue(0);
+                    writer.WriteValue("");
```

Comment:
I don't think this actually writes any bytes/bits - delete?
```suggestion
```

Reply:
If it's already writing and reading the length why are you manually writing the length right above it?

Reply:
Please see Slack discussion